From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Mon, 17 Sep 2018 21:50:15 +0200
Subject: ungoogled-chromium: Disable Network Time Tracker

This connects to Google to check if the system time is correct when a website certificate
date seems incorrect, according to https://bugs.chromium.org/p/chromium/issues/detail?id=725232

Fixes https://github.com/Eloston/ungoogled-chromium/issues/302

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 components/network_time/network_time_tracker.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/components/network_time/network_time_tracker.cc b/components/network_time/network_time_tracker.cc
--- a/components/network_time/network_time_tracker.cc
+++ b/components/network_time/network_time_tracker.cc
@@ -67,6 +67,7 @@ BASE_FEATURE(kNetworkTimeServiceQuerying,
              "NetworkTimeServiceQuerying",
              base::FEATURE_ENABLED_BY_DEFAULT);
 #endif
+SET_CROMITE_FEATURE_DISABLED(kNetworkTimeServiceQuerying);
 
 namespace {
 
@@ -282,7 +283,7 @@ void NetworkTimeTracker::UpdateNetworkTime(base::Time network_time,
 }
 
 bool NetworkTimeTracker::AreTimeFetchesEnabled() const {
-  return base::FeatureList::IsEnabled(kNetworkTimeServiceQuerying);
+  return false;
 }
 
 NetworkTimeTracker::FetchBehavior NetworkTimeTracker::GetFetchBehavior() const {
--
